src/styles/header.scss

```diff
@@ -0,0 +1,58 @@
+@import 'variables';
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+
+  .logo {
+    display: flex;
+  }
+
+  .nav__list {
+    list-style-type: none;
+    display: flex;
+    padding: 0;
+    margin: 0;
+
+    .nav__item {
+      margin-right: 20px;
+      position: relative;
+
+      &:last-child {
+        margin-right: 0;
+      }
+
+      .nav__link {
+        display: flex;
+        align-items: center;
+        height: 60px;
+        text-decoration: none;
+        color: $mainColor;
+        font-size: 12px;
+        font-weight: 500;
+        text-transform: uppercase;
+        position: relative;
+
+        &:hover {
+          color: $active-color;
```

Comment:
The variable `$active-color` is used here but it is not defined in this file. Ensure that it is defined in the '_variables.scss' file and imported correctly.